Makefile

```diff
@@ -31,11 +31,8 @@ GO15VENDOREXPERIMENT := 1
 PKGS := $(shell go list -tags "${DOCKER_BUILDTAGS}" ./... | grep -v ^github.com/docker/distribution/vendor/)
 
 # Resolving binary dependencies for specific targets
-GOLINT_BIN := $(GOPATH)/bin/golint
-GOLINT := $(shell [ -x $(GOLINT_BIN) ] && echo $(GOLINT_BIN) || echo '')
-
-GODEP_BIN := $(GOPATH)/bin/godep
-GODEP := $(shell [ -x $(GODEP_BIN) ] && echo $(GODEP_BIN) || echo '')
+GOLINT := $(shell which golint || echo '')
```

Comment:
Why not just rely on `$PATH`? No reason to fix these outright using `which`, when that will be done automatically. How did this get in here in the first place?